### PR TITLE
Feature#5 crud employer model

### PR DESCRIPTION
**Pull request by willcab**
# Description
add controller employees and correct an error in the model employees

 Examples:
 - What?: I've added support for authentication to implement Key Result 2 of OKR1. It includes
model, table, controller and test. For more background, see ticket #JIRA-123.
 - Why?: These changes complete the user login and account creation experience. See #JIRA-123 for more information.

 # Testing
 Help me how can I test or look at the changes

 Example:
  - I've added coverage for testing all new methods. I used Faker for a few random user emails and names.

  ## Screenshots
If applicable, include screenshots of the results or screenshots that help to see changes

### CODE REVIEW

File: app/app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      redirect_to objects_url
+    end
+  end
+  
+  
+  private
+  def employee_params
+    params.require(:employee).permit(:name, :email, :position, :id_employee, :private_number, :store)
+  end
+end
```
Comment (JuanequeX) [inline app/app/controllers/employees_controller.rb:55]: Add an blank line here at the end of this file

File: app/app/controllers/employees_controller.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    @object = Employee.find(employee_params)
+      if @object.update_attributes(params[:employee])
+        flash[:success] = "Object was successfully updated"
+        redirect_to @object
+      else
+        flash[:error] = "Something went wrong"
+        render 'edit'
+      end
+  end
+  def new
```
Comment (JuanequeX) [inline app/app/controllers/employees_controller.rb:24]: add a extra space between line 23 an 24

File: app/app/controllers/employees_controller.rb [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  
+  def update
+    @object = Employee.find(employee_params)
+      if @object.update_attributes(params[:employee])
+        flash[:success] = "Object was successfully updated"
+        redirect_to @object
+      else
+        flash[:error] = "Something went wrong"
+        render 'edit'
+      end
```
Comment (JuanequeX) [inline app/app/controllers/employees_controller.rb:22]: fix this identation

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by CZamora5 — APPROVED** [review]
Looks ok

**Review by elioflo — APPROVED** [review]
Good Job!

**Review by EduardoBriceno89 — APPROVED** [review]
Looks good. Nice work

File: app/app/controllers/employees_controller.rb (hunk 3 above)
Comment (willcab) [reply app/app/controllers/employees_controller.rb]: add blank lines

File: app/app/controllers/employees_controller.rb (hunk 2 above)
Comment (willcab) [reply app/app/controllers/employees_controller.rb:21]: add blank lines

File: app/app/controllers/employees_controller.rb (hunk 1 above)
Comment (willcab) [reply app/app/controllers/employees_controller.rb]: add blank lines

**Review by JuanequeX — APPROVED** [review]
LGTM ✅

**Pull request merged**